dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/modelWriter.py

```diff
@@ -559,6 +559,7 @@ def fill_setSharedParamsDefault(self):
     def fill_evalFAdept(self):
         self.addEmptyLine()
         self.addLine("#ifdef _ADEPT_\n")
+        self.addBody( self.builder.get_list_for_evalfadept_external_call() )
```

Comment:
At the moment, the omcFunction_adept is written in Dyn_cpp while omcFunction is in external.cpp. Perhaps it would be good to have also the omcFunction_adept in the external.cpp?

Reply:
Solved